pages/agent/index.tsx

```diff
@@ -0,0 +1,238 @@
+          alt="Marché de l'inclusion"
+        />
+        <img src="/images/logos/dila.svg" alt="DILA" />
+
+        <img src="/images/logos/vies.svg" alt="VIES" />
+      </div>
+    </section>
+    <style jsx>
+      {`
+        section.hero {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+        }
+        .hero header {
+          max-width: 500px;
+        }
+        .hero img {
+          width: 400px;
+        }
+
+        section.value img {
+          height: 100px;
+        }
+
+        section {
+          margin-bottom: 4rem;
+        }
+        .logo-soup {
+          display: flex;
+          justify-content: space-between;
+          gap: 2rem;
+          padding: 1rem 0;
+          flex-wrap: wrap;
+        }
+        .logo-soup img {
+          height: 4rem;
+          max-width: 130px;
+        }
+
+        @media (max-width: 768px) {
+          .hero img {
+            display: none;
+          }
+          section {
+            margin-bottom: 2rem;
+          }
+
+          section.value img,
+          section.value h3 {
+            width: 100%;
+            text-align: center;
+          }
+        }
+      `}
+    </style>
+  </>
+);
+
+type ICaseExampleProps = {
+  title: string;
+  description: string;
+  image: string;
+};
+function CaseExample({ title, description, image }: ICaseExampleProps) {
+  return (
+    <>
+      <div className="fr-col-12 fr-col-md-6 case-example">
+        <img src={image} alt="" />
+        <div>
+          <h3>{title}</h3>
+          <p>{description}</p>
+        </div>
+      </div>
+      <style jsx>
+        {`
+          .case-example {
+            display: flex;
+            align-items: center;
+          }
+          img {
+            margin-right: 20px;
+            padding: 4rem;
+            padding-top: 0;
+            padding-bottom: 4rem;
+            margin: 0 -2rem;
+            background: radial-gradient(
+              circle at center,
+              var(--background-alt-blue-france) 0%,
+              var(--background-alt-blue-france) 50%,
+              #ffffff 50%
+            );
+          }
+          img + * {
+            border-left: 2px solid var(--background-alt-blue-france);
+            padding-left: 2rem;
+          }
+          @media (max-width: 768px) {
+            .case-example {
+              flex-wrap: wrap;
+              margin-bottom: 2rem;
+            }
+            img {
+              height: 4rem;
+              padding: 1rem;
+              margin: 0;
+              background: var(--background-alt-blue-france);
+              border-radius: 0.5rem;
+              border-bottom-left-radius: 0;
+            }
+          }
+        `}
+      </style>
+    </>
+  );
+}
+
+LandingPage.getLayout = function getLayout(page: ReactElement) {
+  return <LayoutSimple>{page}</LayoutSimple>;
+};
+
+export const getStaticProps: GetStaticProps = async ({}) => {
+  return {
+    props: {},
+  };
+};
```

Comment:
I am not sure this is necessary. In case there is no GetServerSideProps, Is nt the page static by defautl ?